alot/commands/__init__.py

```diff
@@ -41,6 +55,23 @@ class CommandCanceled(Exception):
     'global': {},
 }
 
+# classes as keys with (mode, cmdname) as values
+REVERSE_COMMANDS = {}
+
+
+def reverse_lookup_command(klass):
+    """
+    returns mode and commandname for a commandclass
+
+    :param klass: the class used to look up the cmdname and mode
+    :type klass: class
+    :rtype: (str, str)
```

Comment:
This is not true, The tuple will be `(None, None)` in the case that there is no defined hook.